exploring_UV_extinction_curves_pt2/Exploring_UV_extinction_curves_pt2.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>I think it's a little confusing to show the results here (in the bullet points) before showing how to access them from Simbad.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>